src/templates/base/2019/base_ebook.html

```diff
@@ -345,30 +345,7 @@ <h1 class="title title-lg">
         <div>
           <span class="contributor-name">{{ contributor.name }}</span>
           <span class="contributor-social">
-          {% if contributor.twitter %}
-            <a href="https://twitter.com/{{ contributor.twitter }}" aria-labelledby="contributors-{{ id }}-twitter">
-              <svg role="img">
-                <title id="contributors-{{ id }}-twitter">{{ onTwitter(contributor.twitter) }}</title>
-                <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#twitter"></use>
-              </svg>
-            </a>
-          {% endif %}
-          {% if contributor.github %}
-          <a href="https://github.com/{{ contributor.github }}" aria-labelledby="contributors-{{ id }}-github">
-            <svg role="img">
-              <title id="contributors-{{ id }}-github">{{ onGitHub(contributor.github) }}</title>
-              <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#github"></use>
-            </svg>
-          </a>
-          {% endif %}
-          {% if contributor.website %}
-          <a href="{{ contributor.website }}" aria-labelledby="contributors-{{ id }}-website">
-            <svg role="img">
-              <title id="contributors-{{ id }}-website">{{ website(contributor.name) }}</title>
-              <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#globe"></use>
-            </svg>
-          </a>
-          {% endif %}
+            {% if contributor.twitter %} (@{{ contributor.twitter }}){% endif %}
```

Comment:
@rviscomi this is the contributors social section!

Reply:
Thanks!

Small design suggestion. What would you think of more distinction between name/Twitter by having them on separate lines, different font size, and/or different shade text?

Similar to how names/handles appear on Twitter itself:

![image](https://user-images.githubusercontent.com/1120896/89376828-a050c680-d6be-11ea-80bd-7c6195120f6e.png)

Reply:
What do you reckon - with Twitter icon or without?

![With Twitter icon](https://user-images.githubusercontent.com/10931297/89406309-e0db2f00-d714-11ea-9b9c-d3b8dc6652d9.png)

![Without Twitter icon](https://user-images.githubusercontent.com/10931297/89406441-0ff1a080-d715-11ea-8b90-99d7f15a29b4.png)

Reply:
Both look great! If I had to pick, the version without.

Reply:
Think the full works is too much?

![All social media with each contributor](https://user-images.githubusercontent.com/10931297/89501194-d1162600-d7ba-11ea-958a-6e836398abf1.png)

Reply:
And with LinkedIn:

![Contributor Social Icons](https://user-images.githubusercontent.com/10931297/89501286-fdca3d80-d7ba-11ea-9719-9d36d468112c.png)

Reply:
I actually like it, even if it is a little busy. It's an appendix anyway.

I also implemented a similar way in the Authors section and removed links when the URL is used:

![Better Author Bios](https://user-images.githubusercontent.com/10931297/89525747-94f4bc80-d7de-11ea-96f2-76f85b05ccfc.png)

I like this so much I'm tempted to change the website to this format as well - WDYT?

Reply:
LGTM. Only nit is to ensure the icons and the text are vertically aligned.

Reply:
They are vertically aligned and look perfect in Chrome, but Prince seems to be adding an offset for some reason. Did try various things to remove it but never got there. Will have another look with a fresh pair of eyes.

Reply:
Fixed it with `vertical-align: text-top;`:

![Author bio](https://user-images.githubusercontent.com/10931297/89733635-ac39e100-da4e-11ea-973f-ebd234c4f879.png)